contracts/truefi2/TrueFiPool2.sol

```diff
@@ -662,6 +662,19 @@ contract TrueFiPool2 is ITrueFiPool2, IPauseableContract, ERC20, UpgradeableClai
         return liquidValue().mul(BASIS_PRECISION).div(_poolValue);
     }
 
+    /**
+     * @dev Ratio of liquid assets in the pool after lending
+     * @param amount Amount of asset being lent
+     * @return Calculated ratio in basis points
+     */
+    function proFormaLiquidRatio(uint256 amount) external override view returns (uint256) {
+        uint256 _poolValue = poolValue();
+        if (_poolValue == 0) {
+            return 0;
+        }
+        return liquidValue().sub(amount).mul(BASIS_PRECISION).div(_poolValue);
```

Comment:
Maybe additional brackets for clarity? Or some other sort of refactoring, like extracting `liquidValue().sub(amount)` to helper variable?
```suggestion
        return (liquidValue().sub(amount)).mul(BASIS_PRECISION).div(_poolValue);
```